tests/tracing/test_pruning.py

```diff
@@ -1098,50 +1137,6 @@ def test_prune_by_length(
             id="Pruning by length and height disabled",
             # marks=pytest.mark.skip(),
         ),
-        pytest.param(
```

Comment:
Why has this set of parameters been removed? Is it because the ability to prune by a fixed (or user specified) length has (or will be) removed?

If so has that been removed from the code base?

Reply:
Default -1 length flag has been changed to 10. Also the ability to use -1 has been removed

Hence this parameterised test is no longer needed as it checked the -1 functionality (15% of length)

Reply:
Is this change pending?

Currently on `maxgamill-sheffield/800-better-tracing` the default value for `max_length` is `None` when the [class is initalised](https://github.com/AFM-SPM/TopoStats/blob/33274043df8c6beedad0b89f60e9e97f378c27ef/topostats/tracing/pruning.py#L240). I recall there was at some point something along the lines of...

```python
self.max_length = max_length if max_length is None else length * 0.15
```

...and I suggested that it be made configurable, which it now is, but the default isn't 10%, its currently `None`.

Reply:
Ahh right, yes it no longer works as a percentage, that is the "feature" that was dropped in favour of a physical value (nm). So `-1` which meant 15% of total length is now `10` for 10nm

The line you reference shows it's default initialisation value can be `None` which is allowed as it will just skip the `prune_by_length` function, I'll re-add the default `None` test then :).

Reply:
Tell a lie - The test before it tests `None` for height and length pruning so I still don't think we need the test.